en/upgrade/centreon-ha/upgrade-from-20.04.md

```diff
@@ -0,0 +1,390 @@
+---
+id: upgrade-centreon-ha-from-20-04
+title: Upgrade Centreon HA from Centreon 20.04
+---
+
+This chapter describes how to upgrade your Centreon HA platform from version 20.10
+to version 21.04.
+
+## Prerequisites
+
+### Suspend cluster resources management
+
+In order to avoid a failover of the cluster during the update, it is necessary to unmanage all Centreon resources, as well as MariaDB.
+
+```bash
+pcs resource unmanage centreon
+pcs resource unmanage ms_mysql
+```
+
+### Perform a backup
+
+Be sure that you have fully backed up your environment for the following
+servers:
+
+- Central server
+- Database server
+
+## Upgrade process
+
+### Update the Centreon repository
+
+Run the following commands:
+
+```shell
+yum install -y http://yum.centreon.com/standard/21.04/el7/stable/noarch/RPMS/centreon-release-21.04-4.el7.centos.noarch.rpm
+```
+
+> **WARNING:** to avoid broken dependencies, please refer to the documentation of the additional modules to update the Centreon Business Repositories.
+
+### Upgrade the Centreon solution
+
+Clean yum cache:
+
+```shell
+yum clean all --enablerepo=*
+```
+
+Then upgrade all the components with the following command:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--HA 2 Nodes-->
+```shell
+yum update centreon\*
+yum install centreon-ha-web centreon-ha-common
+yum autoremove centreon-ha
+```
+<!--HA 4 Nodes-->
+```shell
+# On the Central Servers
+yum update centreon\*
+yum install centreon-ha-web centreon-ha-common
+yum autoremove centreon-ha
+# On the Database Servers
+yum update centreon\*
+yum install centreon-ha-common
+yum autoremove centreon-ha
+```
```

Comment:
I'm not sure about this choice.

- Pros: we don't copy/past a lot of content in two separate doc (and maintain it)
- Cons: we have a single page to upgrade both 2 and 4 nodes architectures

Also, if we keep it like it is, let's split into two code blocks to make copy/paste easier for lazy user like me :) Comments then become simple instructions before each block.

@omercier what do you think?